data_collection/gazette/spiders/sp_sao_bernardo_do_campo.py

```diff
@@ -0,0 +1,280 @@
+import re
+from datetime import date, datetime
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpSaoBernardoDoCampoSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3548708"
+    name = "sp_sao_bernardo_do_campo"
+    allowed_domains = ["saobernardo.sp.gov.br"]
+    start_date = date(2002, 5, 3)
+    end_date = date.today()
+
+    def start_requests(self):
+        base_url = "https://www.saobernardo.sp.gov.br/web/sbc/todas-as-edicoes"
+        yield scrapy.Request(base_url)
+
+    def parse(self, response):
+        base_files_url = "https://www.saobernardo.sp.gov.br"
+        years = response.xpath(
+            "/html/body/div[5]/div/div/div[1]/div/div[2]/div/div/section/div/div/div/a/text()"
```

Comment:
You could reduce the XPath by using the classes that are above these fields in terms of hierarchy. Like:

```
div.portlet-content div.portlet-content-container div.portlet-body a
```
Apart from keeping the xpath short, it will also help with readability.